### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by MAsimZia** [community/community#141522] [Programming Help]
### Body

Excuse me please
Yesterday I added the Google tag code in the website, but it is showing in the Google analytics that no Google tag detected, so this is also a problem and this script of the Google tag is also shown in the website.
For this reason, the Google tag code that I added in the file yesterday, I went here and deleted the code of the Google tag, but still it is showing in the website.
I have attached the pictures. please tell me how to solve this issue

Please help how to set this website and also tell me that how i add gtag in our website 

![sss](https://github.com/user-attachments/assets/5903cb89-2aa9-4bbb-af65-b357ae81b517)
![SS](https://github.com/user-attachments/assets/883a2dfb-1ae9-4e56-a3ca-f5acabc8f486)

**Comment by tpmst** [comment]
Hello @MAsimZia 

It seems that you're facing a couple of issues with Google Tag Manager and your website. Here's a step-by-step guide to help you resolve these issues:

## 1. Google Tag Not Detected:

 - **Ensure Placement:** Verify that the Google Tag Manager code is placed immediately after the opening `<head>` tag on every page of your website. This is crucial for proper detection and functionality.
 - **Correct Tag Code:** Double-check that the Google Tag code you have on your website matches exactly with the one provided in your Google Tag Manager account. Any typo or missing part could lead to detection issues.

## 2. Script Visible on Website:

- **Script Tag Closure:** Make sure that the opening `<script>` tag is properly closed before the opening of the closing `</script>` tag. From the screenshot, it seems like the first `<script>` tag right after the "https://www.googletagmanager.com/gtag/js" src link isn't closed. This could lead to rendering the subsequent script content as text on the webpage.

 - Correct Format:
```
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-XXXXX"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-XXXXX');
</script>
```

**Reply by MAsimZia** [reply]
I'm new and i can't understand it so please can you do it for me please?? Please 🥺 
I takes only 5-10 minutes 😔

**Comment by MAsimZia** [comment]
I'm new so I can't understand so please can you do it for me please it takes your precious 5-10 minutes

Only remove this that is showing publicly from website and add it correctly I'll give you access of hostinger

On Oct 15, 2024 3:03 PM, Tom ***@***.***> wrote:

Hello @MAsimZia<https://github.com/MAsimZia>

It seems that you're facing a couple of issues with Google Tag Manager and your website. Here's a step-by-step guide to help you resolve these issues:

1. Google Tag Not Detected:

  *   Ensure Placement: Verify that the Google Tag Manager code is placed immediately after the opening <head> tag on every page of your website. This is crucial for proper detection and functionality.
  *   Correct Tag Code: Double-check that the Google Tag code you have on your website matches exactly with the one provided in your Google Tag Manager account. Any typo or missing part could lead to detection issues.

2. Script Visible on Website:

  *   Script Tag Closure: Make sure that the opening <script> tag is properly closed before the opening of the closing </script> tag. From the screenshot, it seems like the first <script> tag right after the "https://www.googletagmanager.com/gtag/js" src link isn't closed. This could lead to rendering the subsequent script content as text on the webpage.

  *   Correct Format:


<script async src="https://www.googletagmanager.com/gtag/js?id=G-XXXXX"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-XXXXX');
</script>
